examples/reprojection.html

```diff
@@ -0,0 +1,287 @@
+<!DOCTYPE html>
```

Comment:
Did you intend to include this file in this PR?

If so, please link it from the correct part of index.html & change up the commented bits so that the headings are still visible & the unfinished examples use a single block comment.